src/components/function-summary/result.tsx

```diff
@@ -5,11 +5,20 @@ import { Caption, Pre } from '@components/typography';
 import { border } from '@common/utils';
 import { FunctionSummaryClarityValue } from '@components/function-summary/value';
 import { IconAlertTriangle, IconCircleCheck } from '@tabler/icons';
+import { TransactionStatus } from '@common/constants';
 
-export const FunctionSummaryResult = ({ result }: { result: Transaction['tx_result'] }) => {
+export const FunctionSummaryResult = ({
+  result,
+  txStatus,
+}: {
+  result: Transaction['tx_result'];
+  txStatus: string | undefined;
+}) => {
```

Comment:
As we're touching this file, can we set props with an interface:

```suggestion
interface FunctionSummaryResultProps {
  result: Transaction['tx_result'];
  txStatus: string | undefined;
}
export const FunctionSummaryResult = ({ result, txStatus }: FunctionSummaryResultProps) => {
```